src/lib/V3DutchOrderLib.sol

```diff
@@ -24,12 +24,12 @@ struct V3DutchOrder {
     OrderInfo info;
     // The address which must cosign the full order
     address cosigner;
+    // Adjust the startAmount to account for changes to gas
+    uint256 baseFee;
```

Comment:
nit: something like `startingBaseFee` or `originalBaseFee` could be more descriptive